Sources/Variable.swift

```diff
@@ -74,7 +74,11 @@ public struct Variable : Equatable, Resolvable {
       if let context = current as? Context {
         current = context[bit]
       } else if let dictionary = current as? [String: Any] {
-        current = dictionary[bit]
+        if bit == "count" && dictionary[bit] == nil {
```

Comment:
I am not too sure about this behaviour and worried it could lead to hard to debug problems for template authors when they use `count`. Sometimes `count` would be the number of items in a dictionary, but some other times it could be an item from the dictionary leading to confusion.

Reply:
I think we should go with `count` always, it would always be the count of items in the collection. Then it will be consistent with array, and its going to be more likely you want to get the count of the sequence over the value of count.